backend/src/Edna/Library/Web/Types.hs

```diff
@@ -61,25 +63,52 @@ instance ToSchema ProjectResp where
   declareNamedSchema = gDeclareNamedSchema
 
 -- | Test methodology as submitted by end users.
-data MethodologyReqResp = MethodologyReqResp
-  { mrpName :: Text
-  , mrpDescription :: Maybe Text
-  , mrpConfluence :: Maybe URI
+data MethodologyReq = MethodologyReq
+  { mrqName :: Text
+  , mrqDescription :: Maybe Text
+  , mrqConfluence :: Maybe URI
   } deriving stock (Generic, Show, Eq)
 
-instance Buildable MethodologyReqResp where
+instance Buildable MethodologyReq where
   build = genericF
 
-instance Buildable (ForResponseLog MethodologyReqResp) where
+instance Buildable (ForResponseLog MethodologyReq) where
   build = buildForResponse
 
-deriveJSON ednaAesonWebOptions ''MethodologyReqResp
+deriveJSON ednaAesonWebOptions ''MethodologyReq
 
 -- We define @ToSchema URI@ elsewhere to have less modules
 -- with orphans.
-instance ToSchema URI => ToSchema MethodologyReqResp where
+instance ToSchema URI => ToSchema MethodologyReq where
   declareNamedSchema = gDeclareNamedSchema
 
+-- | Test methodology as response from the server.
+data MethodologyResp = MethodologyResp
+  { mrName :: Text
+  , mrDescription :: Maybe Text
+  , mrConfluence :: Maybe URI
+  , mrProjects :: [Text]
+  } deriving stock (Generic, Show, Eq)
+
+instance Buildable MethodologyResp where
+  build = genericF
+
+instance Buildable (ForResponseLog MethodologyResp) where
+  build = buildForResponse
+
+deriveJSON ednaAesonWebOptions ''MethodologyResp
+
+instance ToSchema URI => ToSchema MethodologyResp where
+  declareNamedSchema = gDeclareNamedSchema
+
+methodologyReqToResp :: MethodologyReq -> [Text] -> MethodologyResp
```

Comment:
As I can see it only used in tests

Reply:
Yes, and it's used in more than one test module. So it's not obvious where to put it. Do you propose to create `Test.Util`?

Reply:
So if you tell me where you think I should put this function, I'll move it there :stuck_out_tongue_winking_eye:

Reply:
So, `Test.Util` sounds better for me. I just was confused what a strange function I see and found that it is used only in tests)